gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/pdeps/ResolveProductDependenciesIntegrationSpec.groovy

```diff
@@ -42,7 +42,8 @@ class ResolveProductDependenciesIntegrationSpec extends IntegrationSpec {
         import ${BaseDistributionExtension.class.getCanonicalName()}
         
         def ext = project.extensions.create("base", BaseDistributionExtension, project)
-        ProductDependencies.registerProductDependencyTasks(project,"runtimeElements", ext);
+        ext.setProductDependenciesConfig(configurations.runtimeClasspath)
```

Comment:
why still creating the custom extension+task vs activating the plugin and using the default task?

Reply:
less boilerplate and it tests the task directly